graphql-dgs-example-shared/build.gradle

```diff
@@ -15,12 +15,14 @@
  */
 
 dependencies {
-    implementation(project(":graphql-dgs-spring-boot-oss-autoconfigure"))
     implementation(project(":graphql-dgs-pagination"))
     implementation(project(":graphql-dgs-extended-scalars"))
     implementation("io.projectreactor:reactor-core")
     implementation("org.springframework:spring-context")
     implementation("org.springframework:spring-web")
     implementation("com.fasterxml.jackson.core:jackson-databind")
     testImplementation ('io.projectreactor:reactor-test')
+    testImplementation(project(":graphql-dgs-spring-graphql-starter-test"))
+    testImplementation(project(":graphql-dgs-spring-graphql-starter"))
+    testImplementation("name.nkonev.multipart-spring-graphql:multipart-spring-graphql:1.+")
```

Comment:
is `name.nkonev.multipart-spring-graphql:multipart-spring-graphql:1.+` really in use?

Reply:
yes - it's a library that adds file uploads, which is what the Spring folks recommend using as well.